### ISSUE TITLE
Failed to install: cannot load incompatible bytecode

### ISSUE THREAD

**Issue by marek22k** [siduck76/NvChad#2438]
**Describe the bug**
```
Error detected while processing /home/bandura/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/bandura/.local/share/nvim/nvchad/bas
e46/defaults: cannot load incompatible bytecode
stack traceback:
        [C]: in function 'dofile'
        /home/bandura/.config/nvim/init.lua:19: in main chunk
Press ENTER or type command to continue
```

**To Reproduce**
1. Run `git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim`

**Expected behavior**
A normal NvChad installation

**Desktop (please complete the following information):**
 - Operating System: Debian-based
 - Terminal: mate-terminal
 - Version of Neovim
 ```
NVIM v0.9.5-dev-6+gf7f5de747
Build type: RelWithDebInfo
LuaJIT 2.1.1692716794
Compilation: /usr/bin/cc -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wvla -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -fno-common -Wno-unused-result -Wimplicit-fallthrough -fdiagnostics-color=always -fstack-protector-strong -DUNIT_TESTING -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/home/bandura/Downloads/neovim/.deps/usr/include/luajit-2.1 -I/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/build/src/nvim/auto -I/home/bandura/Downloads/neovim/build/include -I/home/bandura/Downloads/neovim/build/cmake.config -I/home/bandura/Downloads/neovim/src -I/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include -I/home/bandura/Downloads/neovim/.deps/usr/include

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

**Comment by marek22k** [comment]
Update: The error does not happen with neovim from the Ubuntu PPA, but only when I compile neovim myself and install it as deb.

**Comment by siduck** [comment]
use the LTS version which is 0.9.4 . if its not available on ubuntu then use https://github.com/MordechaiHadad/bob . its better than your PPA

**Comment by marek22k** [comment]
I do not run a PPA myself.

Does this mean that NvChad is not available with self-compiled neovim?

**Comment by siduck** [comment]
@marek22k  it is. although i cant replicate your error, on 0.9.4 and 0.10. Also its a must to use LTS version of neovim for nvchad

**Comment by marek22k** [comment]
Same happens when I install neovim via bob:
```
NVIM v0.9.4
Build type: Release
LuaJIT 2.1.1692716794

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/__w/neovim/neovim/build/nvim.AppDir/usr/share/nvim"

Run :checkhealth for more info
```

**Comment by marek22k** [comment]
Same happens with v8.0 via bob:
```
$nvim -v
NVIM v0.8.2
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-10 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/__w/neovim/neovim/build/cmake.config -I/__w/neovim/neovim/src -I/__w/neovim/neovim/.deps/usr/include -I/usr/include -I/__w/neovim/neovim/build/src/nvim/auto -I/__w/neovim/neovim/build/include
Compiled by root@4ae72dcad031

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/__w/neovim/neovim/build/nvim.AppDir/usr/share/nvim"

Run :checkhealth for more info
```

```
Error detected while processing /home/marek/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/marek/.local/share/nvim/nvchad/base46/defaults: cannot load incompatible bytecode
stack traceback:
        [C]: in function 'dofile'
        /home/marek/.config/nvim/init.lua:19: in main chunk
```

**Comment by siduck** [comment]
@marek22k do a clean install and use only 0.9.4

**Comment by kwiat1990** [comment]
I want to make a clean install on Mac and it gives me the same error.

**Comment by lackovic** [comment]
I had the same problem using `neovim 0.9.1` installed with `snap` on `Ubuntu 22.04.3`. 

I solved it by installing `neovim 0.9.4` using [bob](https://github.com/MordechaiHadad/bob).  

> do a clean install and use only 0.9.4

@siduck The prerequisite of the [quickstart](https://nvchad.com/docs/quickstart/install) mentions `neovim 0.9.0`: should that be updated to require `0.9.4`?

**Comment by siduck** [comment]
> I had the same problem using `neovim 0.9.1` installed with `snap` on `Ubuntu 22.04.3`.
> 
> I solved it by installing `neovim 0.9.4` using [bob](https://github.com/MordechaiHadad/bob).
> 
> > do a clean install and use only 0.9.4
> 
> @siduck The prerequisite of the [quickstart](https://nvchad.com/docs/quickstart/install) mentions `neovim 0.9.0`: should that be updated to require `0.9.4`?

you're right, done.
